ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -62,5 +65,16 @@ export class PipelineListComponent implements OnInit {
     });
   }
 
+  selectChangeHandler (event: any) {
+    //update the ui
+    this.selectedOption = event.target.value;
+    if (this.selectedOption === 'created-date-up'){
```

Comment:
you should sort the original pipelines and rebuild the sections instead of sorting the sections themselves.